Tests/ArgumentParserGenerateDoccReferenceTests/Snapshots/testRollDoccReference().md

```diff
@@ -7,29 +7,31 @@ roll [--times=<n>] [--sides=<m>] [--seed=<seed>] [--verbose]
   [--help]
 ```
 
-- term **--times=\<n\>**:
+### Options
+
+- term **--times=\<n\>:**
 
 *Rolls the dice <n> times.*
 
 
-- term **--sides=\<m\>**:
+- term **--sides=\<m\>:**
 
 *Rolls an <m>-sided dice.*
 
 Use this option to override the default value of a six-sided die.
 
 
-- term **--seed=\<seed\>**:
+- term **--seed=\<seed\>:**
 
 *A seed to use for repeatable random generation.*
 
 
-- term **--verbose**:
+- term **--verbose:**
```

Comment:
Question: does this render as expected? I know that term-definition lists split on the colon but I don't know of the top of my head what happens if the colon is inside a formatted element.